tests/test_acasclient.py

```diff
@@ -437,6 +437,16 @@ def create_and_connect_backdoor_user(self, username = None, password = None, pre
         user_client = acasclient.client(user_creds)
         return user_client
 
+    def verify_file_and_content_equal(self, content, file):
```

Comment:
Can we replace `file` with `fpath` / `file_path` or something on those lines to indicate that this is supposed to be a file path and not a file handle :)